vjudge/models.py

```diff
@@ -0,0 +1,12 @@
+from django.db import models
```

Comment:
This works as a dictionary to find, say, ``UVA id`` to ``vjudge id``.
Otherwise, the old problem will not work.